models/patch.models.js

```diff
@@ -0,0 +1,27 @@
+const db = require('../db/connection')
+const { validateArticleId } = require('../validation-functions/validateArticle-Id')
+
+async function updateVoteByArticleId (article_id, inc_votes) {
+   try {
+    
+   await validateArticleId(article_id)
+    
+   return db.query(
+    `
+    UPDATE articles
+    SET votes = votes + $1
+    WHERE article_id = $2
+    RETURNING *`, 
+    [inc_votes, article_id])
+
+    .then((article)=>{
+        return article
```

Comment:
if you return article.rows[0] here you won't need to return .rows in the controller or check article[0] in the test